app/workers/invalidate_cache_worker.rb

```diff
@@ -0,0 +1,26 @@
+class InvalidateCacheWorker
+  include Sidekiq::Worker
+
+  def perform
+    creds = Aws::ECSCredentials.new(retries: 3)
+    client = Aws::CloudFront::Client.new(credentials: creds)
+
+    production_cdn = client.list_distributions.distribution_list.items.find { |d| d.comment = 'Production CDN' }
```

Comment:
Do we want to parameterise this name via tfvars? I suspect it’ll fail in dev and staging where we might want to validate it

Reply:
I don't mind the job failing in dev/staging, but yeah, this might work better paramed